openquakeplatform_ipt/static/ipt/js/config_files.js

```diff
@@ -1615,15 +1615,22 @@ $(document).ready(function () {
             obj.conditional_loss_poes_choice = $(
                 pfx + ' input[type="checkbox"][name="conditional_loss_poes_choice"]').is(':checked');
 
-            if (obj.conditional_loss_poes_choice) {
-                obj.conditional_loss_poes = $(pfx + ' input[type="text"][name="conditional_loss_poes"]').val();
+            if ((!obj.asset_loss_table) && obj.conditional_loss_poes_choice) {
+                // Was: "To include 'Loss maps' you must enable 'Loss ratios' in 'Risk calculation' section"
+                ret.str += "To include 'Loss maps' you must enable 'Asset loss table' in 'Risk calculation' section.\n";
```

Comment:
'Asset loss table'  -> `asset_loss_table`

Reply:
It is related to the application item, not to the .ini element.